packages/terra-arrange/src/Arrange.jsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames';
+import classNames from 'classnames/bind';
 import 'terra-base/lib/baseStyles';
-import './Arrange.scss';
+import styles from './Arrange.scss';
 
 const alignmentTypes = ['center', 'bottom', 'stretch'];
+const cx = classNames.bind(styles);
```

Comment:
ClassNames has 1st class support for CSS modules. By using bind, we can reference the local class names by name (rather than by key). This allows our code to be more terse in many cases. See https://www.npmjs.com/package/classnames#alternate-bind-version-for-css-modules-